### PR TITLE
Add js strings of module page to translations

### PR DESCRIPTION
**Pull request by Quetzacoalt91**
| Questions | Answers |
| --- | --- |
| Branch? | develop |
| Description? | Woups, many strings in JS were missing from the translated data. This PR add them. |
| Type? | bug fix |
| Category? | BO |
| BC breaks? | Nope |
| Deprecations? | Nope |
| Fixed ticket? | / |
| How to test? | Without the strings being translated, this gonna be difficult to tests them. But at least, they should still appear in English. |

### CODE REVIEW

**Comment by Quetzacoalt91** [comment]
Alexandra d'Even, I need you superpowers on this PR.

**Comment by Quetzacoalt91** [comment]
PR #6698 merged. I rebased this one to make it easier to review.

File: src/PrestaShopBundle/Resources/views/Admin/Module/Includes/menu_top.html.twig [hunk 1]
```diff
@@ -22,3 +22,7 @@
 </div>
 
 <hr class="top-menu-separator"/>
+
+{% set js_translatable = {
+"Search - placeholder": "Search modules: keyword, name, author..."|trans({}, 'Admin.Notifications.Success'),
```
Comment (AlexEven) [inline src/PrestaShopBundle/Resources/views/Admin/Module/Includes/menu_top.html.twig]: Admin.Modules.Help

File: admin-dev/themes/default/js/bundle/module/module.js [hunk 2]
```diff
@@ -605,7 +605,7 @@ var AdminModuleController = function() {
       var menuCategoryToTrigger = $(self.categoryItemSelector+'[data-category-ref="' + refCategory + '"]');
 
       if (!menuCategoryToTrigger.length) {
-        alert('No category with ref ('+refMenu+') seems to exists!');
+        console.warn('No category with ref ('+refMenu+') seems to exists!');
```
Comment (AlexEven) [inline admin-dev/themes/default/js/bundle/module/module.js]: seems to exist!

File: src/PrestaShopBundle/Resources/views/Admin/Module/manage.html.twig [hunk 3]
```diff
@@ -84,3 +84,9 @@
         </div>
     </div>
 {% endblock %}
+
+{% set js_translatable = {
+"Bulk Action - One module minimum": "You need to select at least one module to use the bulk action."|trans({}, 'Admin.Notifications.Success'),
+"Bulk Action - Request not found": "This bulk action request '[1]' does not exist."|trans({}, 'Admin.Notifications.Success'),
+"Bulk Action - Request not available for module": "Action [1] not available for module [2]. Skipped."|trans({}, 'Admin.Notifications.Success'),
```
Comment (AlexEven) [inline src/PrestaShopBundle/Resources/views/Admin/Module/manage.html.twig]: I'd say "The action "[1]" is not available ... "

Thank you, Thomas de Nabord!

**Comment by Quetzacoalt91** [comment]
Pull-request fixed and rebased

**Comment by aleeks** [comment]
Thank you @Quetzacoalt91 I'm waiting for tests!

**Pull request merged**